README.md

```diff
@@ -2,26 +2,53 @@
 
 [![Actions Status](https://github.com/ruby/openssl/workflows/CI/badge.svg)](https://github.com/ruby/openssl/actions?workflow=CI)
 
+OpenSSL for Ruby provides access to SSL/TLS and general-purpose cryptography
+based on the OpenSSL library.
 
-OpenSSL provides SSL, TLS and general purpose cryptography. It wraps the
-OpenSSL library.
+OpenSSL for Ruby is sometimes referred to as "openssl" (in all lowercase) or
+"Ruby/OpenSSL" for disambiguation.
+
+## Compatibility and maintenance policy
+
+OpenSSL for Ruby is released as a RubyGems gem. At the same time, it is part of
+the standard library of Ruby. This is called a [default gem].
+
+Stable branches of OpenSSL for Ruby will receive security and bug fixes for the
+[lifecycle of their corresponding Ruby branch][Ruby Maintenance Branches].
+
+|Branch|Maintenance status             |Ruby compatibility|OpenSSL compatibility                       |
```

Comment:
I was not able to understand what the Branch column meant. Which repository's branch?

Reply:
I see. I was confused. Because the stable (git) branch names are `maint-X.Y`, not `X.Y` written in the document. And there is also no `maint-3.2` yet. If you use the X.Y for your intent, I feel that the column name is "Version of OpenSSL for Ruby" rather than "Branch".

Reply:
That makes sense. However I wasn't able to fit "Version of OpenSSL for Ruby" or "OpenSSL for Ruby" without creating line-wrapping, so I just used "Version" instead. Do you think this is clear enough?

Reply:
I see. Thanks for explaining the context. And yes, the "Version" is clear enough to me.